student.rb

```diff
@@ -1,9 +1,10 @@
 require_relative './person'
+require_relative './classroom'
 
 class Student < Person
   def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
     super(id, age, name, parent_permission)
-    @classroom = classroom
+    self.classroom = classroom
```

Comment:
- [ ] Please create a method - call it **classroom** if you like - that serves as the `belongs-to` relationship between student and classroom. You can refer to this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) if you need help